mux/dispatch.c

```diff
@@ -448,8 +448,8 @@ dropconn(struct sock_active * S)
 /**
  * dispatch_init(socks, nsocks, Q, maxconn):
  * Initialize a dispatcher to accept connections from the listening sockets
- * ${socks}[0 .. ${nsocks} - 1] (but no more than ${maxconn} at once) and
- * shuttle requests/responses to/from the request queue ${Q}.
+ * ${socks[0]} ... ${socks[${nsocks} - 1]} (but no more than ${maxconn} at
```

Comment:
`${socks[${nsocks} - 1]}` should be `${socks[nsocks - 1]}`.  Think of `${foo}` as meaning "interpret this as C code".